db/schema.rb

```diff
@@ -233,7 +233,7 @@
     t.boolean "enable_subscriptions", default: false, null: false
     t.integer "business_address_id"
     t.boolean "show_customer_names_to_suppliers", default: false, null: false
-    t.string "visible", limit: 255, default: "public", null: false
+    t.string "visible", default: "only_through_links", null: false
```

Comment:
I guess a later version of Rails doesn't specify a default limit anymore.
We could actually set a much smaller limit, but I don't know how much this helps optimise the db. Probably not worth it.